### DISCUSSION TITLE
Redirect parameters like $S and $U

### DISCUSSION THREAD

**Discussion by IllDepence** [proginosko/LeechBlockNG#378] [Ideas]
Hi,

I was searching for documentation on parameters to use in redirect parameters like `$S` and `$U` in the default redirect URL but could not find anything besides sections in the source code. E.g.: https://github.com/proginosko/LeechBlockNG/blob/e94ae9838547f703e6ed4efcfcff1533b78a36fc/common.js#L9

In the end I would like to do something like match a URL with a regex `^https?:\/\/(?!old\.)[a-z.]*reddit\.com(.*)$` and then redirect to `https://old.reddit.com$match_group_1`. Or if currently `$U` can be manipulated in some way do a simpler matching and then redirect to `https://old.reddit.com$U_but_without_host`.

Is this possible? Or are there other parameters like `$U` except for the domain, path, and fragment? If not, they (and/or a parameter for a match group in a regex) would be handy. :‌‌)

Thanks in advance for any info.

**Comment by proginosko** [comment]
Currently the only supported parameters are $U (URL of blocked page), $S (block set number), and $K (keyword that triggered the block, if there was one).

Perhaps the neatest and most flexible solution to your request would be to allow the user to specify (probably under Advanced Options!) a regular expression to match against the URL and allow numbered variables within the URL of the blocking page ($1, $2, etc.). I don't think this would be too hard to implement.

**Reply by T-Dane** [reply]
Is there a way to display an image of file that isn't hosted on a site?
